plugins/sslservices/sslservices.cpp

```diff
@@ -606,23 +597,13 @@ OPENSSL_API void OPENSSL_CALL pkRSASeal(ICodeContext *ctx, size32_t & __lenResul
 
             // Cleanup
             EVP_CIPHER_CTX_free(encryptCtx);
-            for (size_t x = 0; x < publicKeys.size(); x++)
-                rtlFree(encryptedKeys[x]);
-            rtlFree(encryptedKeys);
+            delete [] encryptedKeys;
```

Comment:
You still need to iterate and delete the encryptedKeys

Reply:
Fixed.